### DISCUSSION TITLE
Credentials in CI/CD when `AWS_SECRET_ACCESS_KEY` not allowed?

### DISCUSSION THREAD

**Discussion by mikestopcontinues** [aws/aws-cdk#20862] [Q&A]
I can pass env vars into my CI/CD pipeline, but I can't use the default `AWS_` names. 

What are my options for passing the my keys into CDK?

**Comment by denisdifazio** [comment]
> Credentials and region may be specified using environment variables or in configuration files. These are the same variables and files used by other AWS tools such as the AWS CLI and the various AWS SDKs. The CDK Toolkit looks for this information in the following order.
> 
> - The AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_DEFAULT_REGION environment variables. Always specify all three variables, not just one or two.
> - A specific profile defined in the standard AWS config and credentials files, and specified using the --profile option on cdk commands.
> - The [default] section of the standard AWS config and credentials files.


Reference: https://docs.aws.amazon.com/cdk/v2/guide/cli.html#cli-environment

I don't know if CDK works exactly the same as SDK as told in the documentation, but the latter has a fourth option: [load from a JSON file on disk](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-json-file.html)

**Reply by mikestopcontinues** [reply]
The trouble is that I can't use those specific env vars, nor can I create a profile file. From what I can see in CDK, it doesn't have a json config arg.

I can reassign the vars on synth, but I'm hoping there's a more elegant way.
